tests/test_tensorflow_sparse_data_api.py

```diff
@@ -219,19 +214,88 @@ def test_sparse_except_with_diff_number_of_x_y_rows(self):
+                    model = create_model(
+                        input_shape=input_shape, num_of_classes=NUM_OF_CLASSES
+                    )
+                    # Add one extra row on X
+                    dataset_shape_x = (ROWS, INPUT_SHAPES[0])
+                    dataset_shape_y = (ROWS, (NUM_OF_CLASSES,))
+
+                    spoiled_data = create_sparse_array_one_hot_2d(*dataset_shape_x)
+                    spoiled_data[np.nonzero(spoiled_data[0])] = 0
+
+                    ingest_in_tiledb(
+                        uri=tiledb_uri_x,
+                        data=spoiled_data,
+                        batch_size=BATCH_SIZE,
+                        sparse=True,
+                    )
+                    ingest_in_tiledb(
+                        uri=tiledb_uri_y,
+                        data=create_sparse_array_one_hot_2d(*dataset_shape_y),
+                        batch_size=BATCH_SIZE,
+                        sparse=False,
+                    )
+
+                    with tiledb.open(tiledb_uri_x) as x, tiledb.open(tiledb_uri_y) as y:
+                        tiledb_dataset = TensorflowTileDBSparseDataset(
+                            x_array=x, y_array=y, batch_size=BATCH_SIZE
+                        )
+                        with self.assertRaises(Exception):
+                            model.fit(tiledb_dataset, verbose=0, epochs=2)
+
+    def test_except_with_multiple_nz_value_record_of_batch_x_y_rows(self):
+        for input_shape in INPUT_SHAPES:
+            with self.subTest():
+                with tempfile.TemporaryDirectory() as tiledb_uri_x, tempfile.TemporaryDirectory() as tiledb_uri_y:
+                    model = create_model(
+                        input_shape=input_shape, num_of_classes=NUM_OF_CLASSES
+                    )
+                    # Add one extra row on X
+                    dataset_shape_x = (ROWS, input_shape)
+                    dataset_shape_y = (ROWS, (NUM_OF_CLASSES,))
+
+                    spoiled_data = create_sparse_array_one_hot_2d(*dataset_shape_x)
+                    spoiled_data[0] += 1
+
+                    ingest_in_tiledb(
+                        uri=tiledb_uri_x,
+                        data=spoiled_data,
+                        batch_size=BATCH_SIZE,
+                        sparse=True,
+                    )
+                    ingest_in_tiledb(
+                        uri=tiledb_uri_y,
+                        data=create_sparse_array_one_hot_2d(*dataset_shape_y),
+                        batch_size=BATCH_SIZE,
+                        sparse=False,
+                    )
+
+                    with tiledb.open(tiledb_uri_x) as x, tiledb.open(tiledb_uri_y) as y:
+                        tiledb_dataset = TensorflowTileDBSparseDataset(
+                            x_array=x, y_array=y, batch_size=BATCH_SIZE
+                        )
+                        model.fit(tiledb_dataset, verbose=0, epochs=2)
```

Comment:
No assertion here.

Reply:
Test should be successful and not raise an exception